runtime/runtime/src/lib.rs

```diff
@@ -1254,6 +1255,7 @@ impl Runtime {
                 receipt_id = %receipt.receipt_id,
                 node_counter = ?state_update.trie.get_trie_nodes_count())
             .entered();
+            tracing::trace!(target: "io_tracer", predecessor = %receipt.predecessor_id, receiver = %receipt.receiver_id, id = %receipt.receipt_id);
```

Comment:
This makes sense and I updated it accordingly. Now I generally include span fields as well. (With the exception of those that I noticed being formatted strings rather than simple key-value pairs, which were too verbose for human readability and would be hard to parse for computers.)